types/react-loadable/index.d.ts

```diff
@@ -0,0 +1,118 @@
+// Type definitions for react-loadable 3.3
+// Project: https://github.com/thejameskyle/react-loadable#readme
+// Definitions by: Diogo Franco <https://github.com/Kovensky>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import * as React from 'react';
+
+// tslint:disable-next-line strict-export-declare-modifiers
+type LoadedComponent<Props> = React.ComponentClass<Props> | React.SFC<Props>;
```

Comment:
Hm, it is indeed.

I'll just export this one; but is there any way to make an unexported local alias?

This alias also looks helpful enough to maybe belong on react.d.ts.